### ISSUE TITLE
is it possible to get less than 2.0 loss when pretraining from scratch on 20gb dataset on the first 30k steps?

### ISSUE THREAD

**Issue by WissamAntoun** [google-research/bert#968]
Hey, I'm trying to pretrain BERT from scratch on a new language, in my first trial I got a ~2 loss and a 60% MLM accuracy after 800k to 1M steps.

I tried a new way to segment the text then trained a new vocab model, then generated new pretraining data. Now the loss after just 10k steps is 2.7 then drops to below 2 from 30k onwards.

i checked the pretraining data and it seems correct, the MLM accuracy is above 64% on all checkpoints between 10k and 100k.

Is this even possible, and should i continue the training till 1M?

**Comment by RinaldsViksna** [comment]
Loss on my model looks like this, but my dataset is only 6GB, so I think  your loss may be ok. How did you obtain MLM accuracy?
 ![image](https://user-images.githubusercontent.com/25202618/72167467-d8678380-33d3-11ea-8470-a70fdae59698.png)

**Comment by fajri91** [comment]
Hi, I am training only 2millon docs (2GB). Why do I get weird training loss for > 900k steps?
 My training config:
> --train_batch_size=8 
>   --max_seq_length=512 
>   --max_predictions_per_seq=75 
>   --num_train_steps=7000000 
>   --num_warmup_steps=70000 
>   --learning_rate=1e-4
> 

![Screenshot from 2020-03-02 07-31-32](https://user-images.githubusercontent.com/12390478/75633319-ef01bc80-5c57-11ea-8cff-63cfb9f78d85.png)
